pkg/minikube/download/download.go

```diff
@@ -60,7 +61,7 @@ func CreateDstDownloadMock(src, dst string) error {
 // download is a well-configured atomic download function
 func download(src string, dst string) error {
 	var clientOptions []getter.ClientOption
-	if out.IsTerminal(os.Stdout) {
+	if out.IsTerminal(os.Stdout) && !(detect.GithubActionRunner()) {
```

Comment:
```suggestion
	if out.IsTerminal(os.Stdout) && !detect.GithubActionRunner() {
```
The logic is fine, but you don't have to have `()` around the func, so lets remove those to stay consistent with the rest of the code base and after that looks good to merge to me!